[13.0][FIX]Multicompany behaviour fix

SkiBY opened the pull request:
As we have company_dependent attribute, when we don't force context change, there will can be an error if you use multicompany.

All the convertions if user is in another active company will be done by 'current' company and don't like it should be via attributes.

MiquelRForgeFlow reviewed (approved):
LGTM

MiquelRForgeFlow commented:
@alexey-pelykh

alexey-pelykh reviewed (changes requested):
Please drop the version change (it will happen automagically in any case, as well as readme regeneration) and tidy up the commit message, otherwise LGTM. Since readme is regenerated automatically, one should add oneself to `currency_rate_inverted/readme/CONTRIBUTORS.rst`

alexey-pelykh commented:
Don't forget to squash the commits manually :)

SkiBY commented:
@alexey-pelykh done
with code coderage i'm a little bit confused, Should I write some test to increase it somehow?

alexey-pelykh commented on currency_rate_inverted/readme/CONTRIBUTORS.rst:
```diff
@@ -3,7 +3,6 @@
-
-  * Alexey Pelykh <alexey.pelykh@corphub.eu>
-
+* Alexey Pelykh <alexey.pelykh@corphub.eu
```
That change removes nested-list formatting. You may use http://rst.ninjs.org/ to evaluate the file before and after

alexey-pelykh commented on currency_rate_inverted/README.rst:
```diff
@@ -23,7 +23,7 @@ Currency Rate Inverted
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```
Just revert the file entirely

MiquelRForgeFlow replied:
no problem about this, OCA-git-bot will amend it anyways

alexey-pelykh replied:
That is also true. This is a non-blocking request :) Question is only about purity, so no strong feelings about this

SkiBY replied:
it was done by precommit. I can revert if it is needed

alexey-pelykh commented:
Hm, I guess codecov/project is safe to ignore in this case.

SkiBY committed:
As we have company_dependent attribute, when we don't force context change, there will can be an error if you use multicompany.
All the convertions if user is in another active company will be done by 'current' company and don't like it should be via attributes.

SkiBY commented:
hey. could someone merge it probably?

dreispt commented:
/ocabot merge patch
